structs.go

```diff
@@ -1102,3 +1103,38 @@ const (
 
 	ErrCodeReactionBlocked = 90001
 )
+
+const (
+	IntentsGuilds = 1 << iota
+	IntentsGuildMembers
+	IntentsGuildBans
+	IntentsGuildEmojis
+	IntentsGuildIntegrations
+	IntentsGuildWebhooks
+	IntentsGuildInvites
+	IntentsGuildVoiceStates
+	IntentsGuildPresences
+	IntentsGuildMessages
+	IntentsGuildMessageReactions
+	IntentsGuildMessageTyping
+	IntentsDirectMessagesMessages
+	IntentsDirectMessageReaction
```

Comment:
I don't mean to nit-pick too much, but this should probably be plural to match both the docs and `IntentsGuildMessageReactions`.

Reply:
Great catch!